### ISSUE TITLE
No custom webpack config webpack.config.dev.js found in dirs CWD;

### ISSUE THREAD

**Issue by crup** [electrode-io/electrode#1458]
I created two custom webpack config files - `archetype/config/webpack/webpack.config.dev.js` and `archetype/config/webpack/webpack.config.dev.js` and yarn start/clap dev/clap build or any of those commands are unable to pick the custom configuration. 

electrode-archetype-react-app/electrode-archetype-react-app-dev: 6.5.18

## Type of issue
Bug

## My environment

* *OS version/details*: `MacOS Catalina - 10.15.1`
* *Node version:* `v12.12.0`
* *npm version:* `6.13.1`
* *Version of yo :* `3.1.0`

## Expected behavior
Build commands should pick the webpack config from archetype and merge it with existing configurations as it did in previous electrode versions.

## Current behavior
I am getting this message in terminal
```
[wds] info: No custom webpack config webpack.config.dev.js found in dirs CWD; CWD/archetype/config/webpack
```

## Command line output

```
yarn run v1.19.1
$ if test "$NODE_ENV" = "production"; then npm run prod-start; else clap dev; fi
[33mwarn[39m: It looks like you are using yarn.
[33mwarn[39m: Electrode uses optionalDependencies and yarn doesn't handle them like npm.
[33mwarn[39m: While it probably works, there may be unexpected issues.
[08:07:03] xclap version 0.2.39 at ./node_modules/xclap
[08:07:03] NodeJS version v12.12.0 at /usr/local/Cellar/node/12.12.0/bin/node
[08:07:03] CWD is /Users/joshi/Sites/test-project
[08:07:04] Loaded CWD/xclap.js
[08:07:04] PATH already contains ./node_modules/.bin
[08:07:04] NOTE: finally hook is unreliable when stopOnError is set to full
[08:07:04] Process dev's dependency serial array [".remove-log-files",".development-env",".mk-dist-dir",".build.client.babelrc",".build.server.babelrc"]
[08:07:04] -Execute electrode/.remove-log-files as function
[08:07:04] >Done Execute electrode/.remove-log-files as function (1 ms)
[08:07:04] .Execute electrode/.development-env as function
[08:07:04] >Done Execute electrode/.development-env as function (0 ms)
[08:07:04] -Execute electrode/.mk-dist-dir as function
[08:07:04] >Done Execute electrode/.mk-dist-dir as function (0 ms)
[08:07:04] .Execute electrode/.build.client.babelrc as function
[08:07:04] >Done Execute electrode/.build.client.babelrc as function (1 ms)
[08:07:04] -Execute electrode/.build.server.babelrc as function
[08:07:04] >Done Execute electrode/.build.server.babelrc as function (1 ms)
[08:07:04] Done Process dev's dependency serial array [".remove-log-files",".development-env",".mk-dist-dir",".build.client.babelrc",".build.server.babelrc"] (6 ms)
[08:07:04] Execute electrode/dev as function
[08:07:04] -Process electrode/dev.fR concurrent array [".set.css-module.env",".webpack-dev",["server-admin","generate-service-worker"]]
[08:07:04] ..Execute electrode/.set.css-module.env as function
[08:07:04] --Execute electrode/.webpack-dev as function
[08:07:04] ..Process electrode/dev.fR.C concurrent array ["server-admin","generate-service-worker"]
[08:07:04] >>Done Execute electrode/.set.css-module.env as function (2 ms)
[08:07:04] >>Done Execute electrode/.webpack-dev as function (1 ms)
[08:07:04] ---Execute electrode/server-admin as function
[08:07:04] ...Execute electrode/generate-service-worker as function
[08:07:04] >>>Done Execute electrode/generate-service-worker as function (2 ms)
[08:07:04] ----Execute node "./node_modules/electrode-archetype-react-app-dev/lib/dev-admin"  --exec "./node_modules/electrode-archetype-react-app/support/babel-run" --ext js,jsx,json,yaml,log,ts,tsx --watch config src/server -- src/server
[38;2;255;165;0mStarting Electrode webpack dev server[39m
[90m[7m[wds][27m[39m [32minfo[39m: No custom webpack config webpack.config.dev.js found in dirs CWD; CWD/archetype/config/webpack
[90m[7m[wds][27m[39m [32minfo[39m: Using custom entry config from ./src/client/entry.config.js
[90m[7m[wds][27m[39m isomorphic-loader webpack plugin: removed existing config file
[90m[7m[wds][27m[39m [32minfo[39m: Not disabling NodeSourcePlugin.  NODE_ENV: 'development'
[90m[7m[wds][27m[39m [32minfo[39m: Composed webpack config from profiles: _base, _dev, user
[90m[7m[wds][27m[39m [32minfo[39m: Webpack config partials: _base-options, _entry, _subapp-chunks, _output, _resolve, _resolve-loader, _babel, _extract-style, _fonts, _images, _stats, _isomorphic, _pwa, _loadable, _dev-mode, _dev, _html-reporter, _dll-load, _node
[90m[7m[wds][27m[39m [32mexpress webpack dev server listening on port 2992[39m
[2K
[90m[7m[wds][27m[39m webpack bundle is now [32mVALID[39m
[38;2;255;165;0mStarting your app server[39m
[32minfo[39m: APP_SRC_DIR set to src/

App listening on port: 3000
[38;2;255;165;0mWARNING: Did not receive start event from your app server - assuming it started.[39m
 > isomorphic-loader extend require: config is now VALID - webpack dev server mode < 
     ┌──────────────────────────────────────────────────────────────┐
     │              [32m[7m   Electrode Dev Admin Console   [27m[39m               │
     │                                                              │
     │ [37m[7mFor your app server[27m[39m                                          │
     │   [35mA[39m - Restart [35mD[39m - [36minspect-brk[39m mode [35mI[39m - [36minspect[39m mode [35mK[39m - Kill │
     │ [37m[7mFor Electrode webpack dev server[27m[39m  [90m[7m[wds][27m[39m                      │
     │   [35mW[39m - Restart [35mE[39m - [36minspect-brk[39m mode [35mR[39m - [36minspect[39m mode [35mX[39m - Kill │
     │ [35mM[39m - Show this menu [35mQ[39m - Shutdown                              │
     └──────────────────────────────────────────────────────────────┘

```

**Comment by crup** [comment]
Downgrading `electrode-archetype-react-app` and `electrode-archetype-react-app-dev` to  any version up to `6.5.11` works as usual.

**Comment by jchip** [comment]
fixed in 6.5.20
